docs/topics/resources/vertx-starter-example-pom.xml

```diff
@@ -12,8 +12,8 @@
   <description>Example application using RHOAR Vert.x</description>
 
   <properties>
-    <vertx.version>3.4.2.redhat-009</vertx.version>
-    <vertx-maven-plugin.version>1.0.13</vertx-maven-plugin.version>
+    <vertx.version>{VertXVersion}</vertx.version>
+    <vertx-maven-plugin.version>{VertXMvnPluginVersion}</vertx-maven-plugin.version>
```

Comment:
I see `VertXVersion` defined in `document-attributes.adoc`, but where is `VertXMvnPluginVersion` defined?

Reply:
@Ladicek Thank you for calling this out. I've actually noticed that myself a while ago and added the `VertxMvnPluginVersion` variable to`document attributes`, which store the VMP version.

Reply:
Cool, thanks!